app/helpers/static_pages_helper.rb

```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```

Comment:
delete this file because you are not using it

Reply:
Done